### PR TITLE
Fifth version of my portfolio at Microverse

### PR DESCRIPTION
**Pull request by Yothu**
This is the "Contact me" section of the mobile version of my portfolio.

The files are:

- index.html
- css/style.css
- linters for HTML/CSS
- background images
- social media images
- miscellaneous images
- README.md
- MIT.md
- .gitignore

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,10 +202,47 @@ <h3>Skills</h3>
...
+    </p>
+    <form class="data-form" action="https://formspree.io/f/mjvlbpay" method="post">
+      <label for="contact-name">
+        <input type="text" class="contact-form" id="contact-name" name="contact-name" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label for="contact-email">
+        <input type="email" class="contact-form" id="contact-email" name="contact-email" placeholder="Email address" required>
+      </label>
+      <textarea class="contact-form" id="contact-message" name="contact-message" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit" id="submit">Get in touch</button>
```
Comment (MrBrN197) [inline index.html:223]: - [x] Your design looks really good. well done 👍. I'd like to point out however that the button on the Figma template doesn't have a border around the button. please try to fix this so your button matches the Figma template perfectly 🙏. 
## Your Design
image
## Figma Design 
image

Comment (MrBrN197) [inline index.html:223]: - [x] Hi the rest of your form design matches the Figma really well 👏. Although I'd like to mention that you have missed including a background image that is Visible on the Figma template. Please implement this
## Your Design
image
## Figma Design
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @Yothu  & @eduardosancho 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Really good form section design  👍.
- ✔Good clean code 👍.
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (andersonlebon) [inline index.html:223]: - [x] Job job :+1:  :clap:  on making the changes requested by the previous reviewer. However, I can still see the same implementation
Screenshot from 2021-12-01 01-48-07

Kindly you may remove that border around the button by using the property `border: none` in the `#submit` attribute in your CSS.

File: index.html [hunk 2]
```diff
@@ -197,10 +202,47 @@ <h3>Skills</h3>
...
+    <p class="get-touch">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form class="data-form" action="https://formspree.io/f/mjvlbpay" method="post">
+      <label for="contact-name">
+        <input type="text" class="contact-form" id="contact-name" name="contact-name" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label for="contact-email">
+        <input type="email" class="contact-form" id="contact-email" name="contact-email" placeholder="Email address" required>
+      </label>
```
Comment (andersonlebon) [inline index.html:221]: - [x] Buy following the requirements and the suggestion of the previous reviewer this form has a background image. check the image bellow :point_down:
Screenshot from 2021-12-01 01-47-20

- [x]  Kindly let's go ahead and add that background image.

**Review by andersonlebon — CHANGES_REQUESTED** [review]

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Yothu  

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments above :up: :arrow_up:  the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @Yothu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**